optimum/onnxruntime/model.py

```diff
@@ -0,0 +1,111 @@
+#  Copyright 2021 The HuggingFace Team. All rights reserved.
```

Comment:
ORTModel has some overlaps with ORTTrainer, we might go for it in this PR and then merge it with ORTTrainer in PR#77!